src/pages/priority/components/UpdatePriorityForm.tsx

```diff
@@ -1,100 +0,0 @@
-import { Form, FormInstance, Input, InputNumber, Select } from "antd";
```

Comment:
also why did you delete this?

Reply:
I deleted the UpdatePriorityForm.tsx and RegisterPriorityForm.tsx because those files are from the KAN-12 (proriry tooltips) but I accidentally added them to this KAN so I removed them from this pull request